### PR TITLE
Updated Some More Tweaks

### PR DESCRIPTION
**Pull request by Disk2019**

### CODE REVIEW

**Comment by undergroundwires** [comment]
Awesome, one little indenting issue. Could you move lines 392 + 393 + 394 to left to align them at same level as `code:` in two lines before. Same issue with 528,529,530,531 and 536,537,538,539 just put them on the left little bit so that they are at the same level as the line before.

**Comment by Disk2019** [comment]
Brother after i click resolve conflicts & corrected those issues then where to click to submit the corrections.

**Comment by undergroundwires** [comment]
> Brother after i click resolve conflicts & corrected those issues then where to click to submit the corrections.

It seems like you cannot do it anymore on the GUI. Probably it's because the forked repository does not exist anymore. I'll try to fix it.

File: src/application/application.yaml [hunk 1]
```diff
@@ -388,6 +388,10 @@ actions:
                         name: Deny app sync with devices
                         recommend: true
                         code: reg add "HKLM\SOFTWARE\Policies\Microsoft\Windows\AppPrivacy" /v "LetAppsSyncWithDevices" /t REG_DWORD /d 2 /f
+                    -
+				        name: Disable Sync Provider Notifications
```
Comment (undergroundwires) [inline src/application/application.yaml:392]: Move to "UI for privacy" category

File: src/application/application.yaml [hunk 2]
```diff
@@ -521,10 +525,18 @@ actions:
...
+					        REG ADD "HKCU\Software\Microsoft\Windows\CurrentVersion\Search" /v "RestrictImplicitTextCollection" /d 1 /t REG_DWORD /f
+					        REG ADD "HKCU\Software\Microsoft\Windows\CurrentVersion\Search" /v "RestrictImplicitInkCollection" /d 1 /t REG_DWORD /f
+					        REG ADD "HKCU\SOFTWARE\Microsoft\InputPersonalization\TrainedDataStore" /v "HarvestContacts" /d 0 /t REG_DWORD /f                    
                     -
                         name: Disable web search in search bar
                         recommend: true
-                        code: reg add "HKLM\SOFTWARE\Policies\Microsoft\Windows\Windows Search" /v DisableWebSearch /t REG_DWORD /d 1 /f
+                        code: |-
+                              reg add "HKLM\SOFTWARE\Policies\Microsoft\Windows\Windows Search" /v DisableWebSearch /t REG_DWORD /d 1 /f
+						      REG ADD "HKCU\Software\Microsoft\Windows\CurrentVersion\Search" /v "BingSearchEnabled" /d 0 /t REG_DWORD /f
```
Comment (undergroundwires) [inline src/application/application.yaml:537]: already exists in "Disable cortana"

File: src/application/application.yaml [hunk 3]
```diff
@@ -521,10 +525,18 @@ actions:
...
+						      REG ADD "HKCU\Software\Microsoft\Windows\CurrentVersion\Search" /v "AllowSearchToUseLocation" /d 0 /t REG_DWORD /f
```
Comment (undergroundwires) [inline src/application/application.yaml:538]: already exists in "Disable cortana"

File: src/application/application.yaml [hunk 4]
```diff
@@ -521,10 +525,18 @@ actions:
...
+						      REG ADD "HKCU\Software\Microsoft\Windows\CurrentVersion\Search" /v "CortanaConsent" /d 0 /t REG_DWORD /f
```
Comment (undergroundwires) [inline src/application/application.yaml:539]: move to "Disable cortana"

**Comment by undergroundwires** [comment]
Fixed the issues & merged manually as fork is deleted &

**Pull request closed without merging**